jubilant/_pretty.py

```diff
@@ -73,7 +73,7 @@ def dump(value: object, indent: str = '') -> str:
         return repr(value)
 
 
-def gron(value: object, prefix: str = '') -> Generator[str, None, None]:
+def gron(value: object, prefix: str = '') -> Generator[str]:
```

Comment:
Drive-by: I've discovered the `, None, None` are optional here, which is nice, so I've simplified those.